### DISCUSSION TITLE
Object reflectiveness in AR mode differs between webxr and scene-viewer.

### DISCUSSION THREAD

**Discussion by dvbridges** [google/model-viewer#3982] [Q&A] [answered]
### Description

There are differences in object reflectiveness in AR mode between webxr and scene-viewer. WebXR uses the environment-image provided and has high reflectiveness, whereas scene-viewer appears to have little or no reflection, and does not use the environment image provided. Also similar with quick-look on iOS.

#### Live Demo

https://rg-l-temporal.glitch.me

This example is set to use scene-viewer then quick-look - webxr reflection can be observed in attached images.

### Version

- model-viewer: v2.1.1

### Browser Affected

- [x] Chrome, version: 107.0.5304.141
- [x] Samsung Internet 19.0.1.2

### OS

- [x] Android
- [x] iOS

### AR

- [x] SceneViewer
- [x] QuickLook

![scene-viewer](https://user-images.githubusercontent.com/22459095/205135707-57aeb7c7-78ea-4811-b98c-e5d00862fada.jpg)
^^ scene-viewer

![webxr](https://user-images.githubusercontent.com/22459095/205135701-6f3f5297-883e-40a1-ab76-3a5a6f15899d.jpg)
^^webxr

**Comment by elalish** [comment, marked as answer]
Indeed; ARCore's lighting estimation isn't high-res enough to make shiny things look shiny - this is a problem for Scene Viewer and also for WebXR if you enable `xr-environment`. ARKit does a better job, but generally has a different estimated lighting feel as well. Not much we can do about it from here, sadly.

**Reply by phatpixels** [reply]
I came here looking for a solution to this reflections limitation issue also. Webxr does an ok job with reflections, but we are then missing the snapshot/capture function that scene-viewer provides. Is there a way to overcome this? Thanks.

**Reply by elalish** [reply]
Ah yes, that's been on the list for some time. A PR would be very welcome, not sure when I'll have time to tackle that.
